experiments/notebooks/visualizations/eth_supply_simulator.py

```diff
@@ -166,23 +168,11 @@ def update_validator_adoption_sliders_by_scenarios(validator_dropdown):
 def update_output_graph(validator_adoption, pos_launch_date, eip1559_base_fee):
     df, parameters = run_simulation(validator_adoption, pos_launch_date, eip1559_base_fee)
 
-    if validator_adoption == 3:
-        validator_dropdown = 'Normal Adoption'
-    elif validator_adoption == 3 * 0.5:
-        validator_dropdown = 'Low Adoption'
-    elif validator_adoption == 3 * 1.5:
-        validator_dropdown = 'High Adoption'
-    else:
-        validator_dropdown = 'Custom'
-
-    if eip1559_base_fee == 0:
-        eip1559_dropdown = 'Disabled'
-    elif eip1559_base_fee == 100:
-        eip1559_dropdown = 'Enabled: Steady State'
-    elif eip1559_base_fee == 70:
-        eip1559_dropdown = 'Enabled: MEV'
-    else:
-        eip1559_dropdown = 'Custom'
+    _validator_scenarios = dict((v, k) for k, v in validator_scenarios.items())
```

Comment:
@marthendalnunes I made a minor refactor here to make the code more "DRY" (Don't repeat yourself) - https://x-team.com/blog/principles-clean-code/ . A few other best practices in there I always find useful reviewing occasionally :)